static/d3/d3.v2.js

```diff
@@ -0,0 +1,7026 @@
+(function() {
```

Comment:
This file looks like a library file. Is it correct to copy it verbatim to RAGE (under what license?), or should we be referencing it from the Internet?

Reply:
I'd suggest we just include this with `<script src="https://d3js.org/d3.v4.min.js"></script>` (from https://d3js.org/).

Reply:
Sounds good to me.